_posts/2020-06-18-a-sustainable-nuget-marketplace-will-have-to-compete-with-the-nuget-gallery.markdown

```diff
@@ -0,0 +1,331 @@
+
+### Getting open source projects on board
+
+This NuGet marketplace will help in making open source sustainable, so it will need open souce producers to
+provide their packages here. I think many will do so, but there's an issue that comes to mind...
+
+How will consumers find this NuGet marketplace, and thus, these producers?
+
+If there are no consumers, we're back at the Gitub sponsor button and sustaining Tom with $15/month. That won't work.
+
+So how to go about this?
+
+## Compete head-on with the NuGet.org gallery
+
+Right now, there is one preconfigured feed on every .NET developer's machine: NuGet.org. That's a huge advantage
+in the current world, but not for this NuGet marketplace. People have to configure its feed in their environment.
+What's the incentive to do that?
+
+### NuGet.org must become less valuable
+
+The only incentive as a consumer I see, is being forced to start using the NuGet marketplace.
+And that can only be done when NuGet.org becomes less valuable. If all of the open source packages I use,
+whether free or paid, are on this NuGet marketplace, I will configure the feed to consume them. If NuGet.org keeps
+giving me Automapper, Autofac, IdentityServer and all that, why would I bother with this NuGet marketplace as a consumer?
+
+The NuGet marketplace will have to provide free open source packages as well. Much like in the JVM world folks
+are using both [Maven Central](https://search.maven.org/) and [JCenter](https://bintray.com/bintray/jcenter)
+for their projects, instead of just one central gallery. In the .NET world, we will have to move to a future
+of multiple authoritative package registries, instead of just one.
+
+### Competition will come anyway
+
+Microsoft is in the money-making business, and that's good. They also own [NuGet.org](https://www.nuget.org/),
+[npm](https://www.npmjs.org), and [GitHub](https://www.github.com/). If a NuGet marketplace is executed well
+and proves there is value, the above platforms will come with their own marketplace. This is a given,
+it will happen.
+
+Having a competitior in the NuGet marketplace space means we're going back to fragmentation, and moving away
+from the single point of contact/sales, open source and dual-licensed open source being in a central place,
+and all that. What this will mean for the ecosystem, I can't tell. But since it will happen anyway, I would
+like to see the first NuGet marketplace to compete head-on, and aggressively become the place where open source
+sustainability thrives.
+
+The alternative is not competing head-on, which will mean we end up in a fragmented world, where the NuGet
+marketplace itself as a business may not be sustainable, and ultimately, hurt producer and consumer trust
+in sustainable .NET open source.
+
+## Conclusion
+
+If all of these criteria (and probably more) are met, this could solve open source sustainability. A solid,
```

Comment:
"solve" is a big word..... i'd be more conservative and say "huge step towards open source sustainability"